templates/terraform/environments/prod/user_access.tf

```diff
@@ -0,0 +1,111 @@
+# define AWS policy documents for developer
+data "aws_iam_policy_document" "developer_access" {
+  # IAM
+  statement {
```

Comment:
This should be moved to the module. Added a note there.

Reply:
done